src/renderer/controllers/torrent-list-controller.js

```diff
@@ -128,6 +128,26 @@ module.exports = class TorrentListController {
     }
   }
 
+  pauseAllTorrents () {
+    this.state.saved.torrents.forEach((torrentSummary) => {
+      if (torrentSummary.status === 'downloading') {
+        torrentSummary.status = 'paused'
+        ipcRenderer.send('wt-stop-torrenting', torrentSummary.infoHash)
+        sound.play('DISABLE')
```

Comment:
Could `sound.play` be moved outside the `forEach` loop? This could get funky if there is a large number of torrents.